pkg/runner/common.go

```diff
@@ -0,0 +1,11 @@
+package runner
+
+// Use consistent IP address ranges for both the data and the control subnet.
+// _which_
+var (
+	controlSubnet  = "192.168.0.0/16"
+	controlGateway = "192.168.0.1"
+	// this gives us two public A blocks and a private A block: 8., 9., & 10.
+	dataSubnet  = "8.0.0.0/5"
+	dataGateway = "8.0.0.1"
+)
```

Comment:
I'm hard-coding these IP ranges. Thoughts? This makes it easier to write tests with consistent IP addresses.